.github/workflows/docker-publish.yml

```
@@ -21,7 +21,23 @@ env:
 
 
 jobs:
+  preBuild:
+    runs-on: ubuntu-latest
+    outputs:
+        tag: ${{steps.vars.tag}}
```

actionlint (1.7.4)

property "tag" is not defined in object type {conclusion: string; outcome: string; outputs: {string => string}}

(expression)